### PR TITLE
add(epp): outscale

### PR DESCRIPTION
**Pull request by garnier-quentin**
## Target version

- [ ] 21.10.x (staging)
- [ ] 22.04.x (staging)
- [ ] 22.10.x (staging)
- [ ] Cloud (staging)
- [X] Plugin Packs (staging)
- [ ] 23.04.x (next)

### CODE REVIEW

File: pp/integrations/plugin-packs/procedures/cloud-outscale.md [hunk 1]
```diff
@@ -0,0 +1,360 @@
...
+title: Outscale API
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Pack Assets
+
+### Templates
+
+The Centreon Pack **Outscale** brings a host template:
```
Comment (smau-centreon) [inline pp/integrations/plugin-packs/procedures/cloud-outscale.md:12]:
```suggestion
The Centreon Monitoring Connector **Outscale** brings a host template:
```

File: pp/integrations/plugin-packs/procedures/cloud-outscale.md [hunk 2]
```diff
@@ -0,0 +1,360 @@
...
+}
+```
+
+## Setup
+
+### Monitoring Pack
+
+If the platform uses an *online* license, you can skip the package installation
+instruction below as it is not required to have the pack displayed within the
+**Configuration > Plugin Packs > Manager** menu.
```
Comment (smau-centreon) [inline pp/integrations/plugin-packs/procedures/cloud-outscale.md:230]:
```suggestion
**Configuration > Monitoring Connectors Manager** menu.
```

File: pp/integrations/plugin-packs/procedures/cloud-outscale.md [hunk 3]
```diff
@@ -0,0 +1,360 @@
...
+If the platform uses an *offline* license, install the package on the **central server**
+with the command corresponding to the operating system's package manager:
+
```
Comment (smau-centreon) [inline pp/integrations/plugin-packs/procedures/cloud-outscale.md:233]: EL9?

File: pp/integrations/plugin-packs/procedures/cloud-outscale.md [hunk 4]
```diff
@@ -0,0 +1,360 @@
...
+```bash
+apt install centreon-pack-cloud-outscale
+```
+
+</TabItem>
+</Tabs>
+
+Whatever the license type (*online* or *offline*), install the **Outscale** Pack through
+the **Configuration > Plugin Packs > Manager** menu.
```
Comment (smau-centreon) [inline pp/integrations/plugin-packs/procedures/cloud-outscale.md:259]:
```suggestion
the **Configuration > Monitoring Connectors Manager** menu.
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-outscale.md [hunk 5]
```diff
@@ -0,0 +1,359 @@
...
+title: Outscale API
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Contenu du Pack
+
+### Modèles
+
+Le Pack Centreon **Outscale** apporte un modèle d'hôte :
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-outscale.md:12]:
```suggestion
Le connecteur de supervision Centreon **Outscale** apporte un modèle d'hôte :
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-outscale.md [hunk 6]
```diff
@@ -0,0 +1,359 @@
...
+|:--------------------------------------------|:-----------------------------------------------------------|
+| Cloud-Outscale-Osccli-Client-Gateway-Name   | Découvre les passerelles clientes et supervise le statut   |
+| Cloud-Outscale-Osccli-Internet-Service-Name | Découvre les services internet et supervise le statut      |
+| Cloud-Outscale-Osccli-Load-Balancer-Name    | Découvre les load balancers et supervise le statut         |
+| Cloud-Outscale-Osccli-Nat-Service-Name      | Découvre les services NAT et supervise le statut           |
+| Cloud-Outscale-Osccli-Net-Name              | Découvre les Nets et supervise le statut                   |
+| Cloud-Outscale-Osccli-Quota-Type-Name       | Découvre les quotas et supervise le statut                 |
+| Cloud-Outscale-Osccli-Route-Table-Id        | Découvre les tables de routage et supervise le statut      |
+| Cloud-Outscale-Osccli-Subnet-Name           | Découvre les sous-réseaux et supervise le statut           |
+| Cloud-Outscale-Osccli-Virtual-Gateway-Name  | Découvre les paserrelles virtuelles et supervise le statut |
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-outscale.md:46]:
```suggestion
| Cloud-Outscale-Osccli-Virtual-Gateway-Name  | Découvre les passerelles virtuelles et supervise le statut |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-outscale.md [hunk 7]
```diff
@@ -0,0 +1,359 @@
...
+| vpn_connections.deleting.count  |       |
+| vpn_connections.deleted.count   |       |
+| vpn connection status           |       |
+
+</TabItem>
+</Tabs>
+
+## Prérequis
+
+Veuillez suivre la procédure officielle en réalisation l'installation de ```osc-cli``` pour l'utilisateur ```centreon-engine``` :
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-outscale.md:206]:
```suggestion
Veuillez suivre la procédure officielle pour l'installation de ```osc-cli``` pour l'utilisateur ```centreon-engine``` :
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-outscale.md [hunk 8]
```diff
@@ -0,0 +1,359 @@
...
+    ]
+}
+```
+
+## Installation
+
+### Pack de supervision
+
+Si la plateforme est configurée avec une licence *online*, l'installation d'un paquet
+n'est pas requise pour voir apparaître le pack dans le menu **Configuration > Plugin Packs > Gestionnaire**.
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-outscale.md:229]:
```suggestion
n'est pas requise pour voir apparaître le pack dans le menu **Configuration > Gestionnaire de connecteurs de supervision**.
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-outscale.md [hunk 9]
```diff
@@ -0,0 +1,359 @@
...
+
+Au contraire, si la plateforme utilise une licence *offline*, installez le paquet
+sur le **serveur central** via la commande correspondant au gestionnaire de paquets
+associé à sa distribution :
+
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-outscale.md:234]: EL9 ?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-outscale.md [hunk 10]
```diff
@@ -0,0 +1,359 @@
...
+```bash
+apt install centreon-pack-cloud-outscale
+```
+
+</TabItem>
+</Tabs>
+
+Quel que soit le type de la licence (*online* ou *offline*), installez le Pack **Outscale**
+depuis l'interface web et le menu **Configuration > Plugin Packs > Gestionnaire**.
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-outscale.md:260]:
```suggestion
depuis l'interface web et le menu **Configuration > Gestionnaire de connecteurs de supervision**.
```

**Review by smau-centreon — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-outscale.md [hunk 11]
```diff
@@ -0,0 +1,359 @@
...
+
+<Tabs groupId="sync">
+<TabItem value="Alma / RHEL / Oracle Linux 8" label="Alma / RHEL / Oracle Linux 8">
+
+```bash
+dnf install centreon-pack-cloud-outscale
+```
+
+</TabItem>
+<TabItem value="CentOS 7" label="CentOS 7">
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-outscale.md:243]:
```suggestion
</TabItem>
<TabItem value="Alma / RHEL / Oracle Linux9" label="Alma / RHEL / Oracle Linux 9">

```bash
dnf install centreon-pack-cloud-outscale
```

</TabItem>
<TabItem value="CentOS 7" label="CentOS 7">
```

File: pp/integrations/plugin-packs/procedures/cloud-outscale.md [hunk 12]
```diff
@@ -0,0 +1,360 @@
...
+
+<Tabs groupId="sync">
+<TabItem value="Alma / RHEL / Oracle Linux 8" label="Alma / RHEL / Oracle Linux 8">
+
+```bash
+dnf install centreon-pack-cloud-outscale
+```
+
+</TabItem>
+<TabItem value="CentOS 7" label="CentOS 7">
```
Comment (omercier) [inline pp/integrations/plugin-packs/procedures/cloud-outscale.md:242]:
```suggestion
</TabItem>
<TabItem value="Alma / RHEL / Oracle Linux 9" label="Alma / RHEL / Oracle Linux 9">

```bash
dnf install centreon-pack-cloud-outscale
```

</TabItem>
<TabItem value="CentOS 7" label="CentOS 7">
```

**Review by omercier — CHANGES_REQUESTED** [review]

**Review by omercier — APPROVED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-outscale.md [hunk 13]
```diff
@@ -0,0 +1,362 @@
...
+dnf install centreon-pack-cloud-outscale
+```
+
+</TabItem>
+<TabItem value="Alma / RHEL / Oracle Linux9" label="Alma / RHEL / Oracle Linux 9">
+
+```bash
+dnf install centreon-pack-cloud-outscale
+
+```bash
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-outscale.md:248]: L'onglet Centos 7 a disparu.

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-outscale.md [hunk 14]
```diff
@@ -0,0 +1,362 @@
...
+
+À partir de Centreon 22.04, il est possible de demander le déploiement automatique
+du plugin lors de l'utilisation d'un pack. Si cette fonctionnalité est activée, et
+que vous ne souhaitez pas découvrir des éléments pour la première fois, alors cette
+étape n'est pas requise.
+
+> Plus d'informations dans la section [Installer le plugin](/docs/monitoring/pluginpacks/#installer-le-plugin).
+
+Utilisez les commandes ci-dessous en fonction du gestionnaire de paquets de votre système d'exploitation :
+
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-outscale.md:275]: Onglet EL9 manquant.

**Comment by smau-centreon** [comment]
The topic is not positioned in the right place (in the TOC) 
image

File: pp/integrations/plugin-packs/procedures/cloud-outscale.md [hunk 15]
```diff
@@ -0,0 +1,363 @@
...
+dnf install centreon-pack-cloud-outscale
+```
+
+</TabItem>
+<TabItem value="Alma / RHEL / Oracle Linux 9" label="Alma / RHEL / Oracle Linux 9">
+
+```bash
+dnf install centreon-pack-cloud-outscale
+
+```bash
```
Comment (smau-centreon) [inline pp/integrations/plugin-packs/procedures/cloud-outscale.md:247]: Centos7 tab has been removed.

File: pp/integrations/plugin-packs/procedures/cloud-outscale.md [hunk 16]
```diff
@@ -0,0 +1,363 @@
...
+When this feature is enabled, you can skip the installation part below.
+
+You still have to manually install the plugin on the poller(s) when:
+- Automatic plugin installation is turned off
+- You want to run a discovery job from a poller that doesn't monitor any resource of this kind yet
+
+> More information in the [Installing the plugin](/docs/monitoring/pluginpacks/#installing-the-plugin) section.
+
+Use the commands below according to your operating system's package manager:
+
```
Comment (smau-centreon) [inline pp/integrations/plugin-packs/procedures/cloud-outscale.md:276]: EL9 tab still missing.

**Review by smau-centreon — CHANGES_REQUESTED** [review]

**Review by smau-centreon — APPROVED** [review]

**Review by omercier — APPROVED** [review]

**Pull request merged**